src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,28 @@ public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomeException {
```

Comment:
The method `register` throws `CustomeException`, but there is a typo in the exception name. It should be `CustomException`.